stdlib/LinearAlgebra/src/schur.jl

```diff
@@ -147,26 +147,37 @@ function schur(A::RealHermSymComplexHerm)
     F = eigen(A; sortby=nothing)
     return Schur(typeof(F.vectors)(Diagonal(F.values)), F.vectors, F.values)
 end
-function schur(A::UpperTriangular{T}) where {T}
+function schur(A::Union{UnitUpperTriangular{T},UpperTriangular{T}}) where {T}
```

Comment:
(Should we have an `AbstractTriangular{T,uplo}` type?)

Reply:
Indeed, I was searching for some type like that (or a union type), but couldn't find anything.